index.html

```diff
@@ -40,11 +44,38 @@
                         <li id="section"><strong>IQ Test</strong> / Home Page</li>
                     </ul>
 
-
+    <div>
+        
+        </div>
     <div class="container">
-        <div>
-            <p>Welcome</p>
+        <h1>
+            <span>W</span>
+            <span>e</span>
+            <span>l</span>
+            <span>c</span>
+            <span>o</span>
+            <span>m</span>
+            <span>e</span>
+            <span> </span>
+            <span>t</span>
+            <span>o</span>
+            <span></span>
+            <span>I</span>
+            <span>Q</span>
+            <span></span>
+            <span>T</span>
+            <span>e</span>
+            <span>s</span>
+            <span>t</span>
+        </h1>
+        
+
+        <div>This test has to sections</div>
+        <div id="sections">
+              <div id="sec1">In the first section you will encounter mathematical questions and you need to answer them in a certain amount of time.</div>  
```

Comment:
This explanation is fine, but from a readability stand-point maybe use shorter sentences? Like: "In the first section you will encounter mathematical questions. Answer them in the allotted time." (Or "Please answer them..." if you prefer)